ops/storage-ddls/cassandra-setup.cql

```diff
@@ -140,44 +155,58 @@ CREATE TABLE popularplaces (
     period text,
     pipelinekey text,
     externalsourceid text,
-    placename text,
     placeid text,
```

Comment:
Should this be a `Place` type so that we don't have to do another trip to the DB and fetch the place info?